api/src/main/proto/input/reqreply/message/create.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package rys.ajaxpetproject.request.message.create.proto;
+
+import "commonmodels/message/message.proto";
+import "commonmodels/error/error.proto";
+
+option java_multiple_files = true;
+
+message MessageCreateRequest {
+  commonmodels.message.proto.Message message = 1;
+  string chatId = 2;
+}
+
+message MessageCreateResponse {
+
+  oneof response {
+    Success success = 1;
+    Failure failure = 2;
+  }
+
+  message Success {
+    string result = 1;
```

Comment:
what is purpose of this string?

Reply:
Give some success message back. 
Not necessary but logic was something like : "MessageService saved message and tells what he saved BUT I don't want Client to see other fields in Message"

Reply:
Success is by it self marker that create was finished successfully, message there is redundant

Reply:
Fair enough, thanks